website/datafuse/mkdocs.yml

```diff
@@ -170,6 +170,11 @@ nav:
       - Conversion Functions:
           - CAST: sqlstatement/conversion-functions/cast.md
           - Type Conversion: sqlstatement/conversion-functions/type-conversion.md
+      - Date and Time Functions:
```

Comment:
Is there any pattern matching to update the contents automatically? I did not see these in clickhouse docs.  https://github.com/ClickHouse/ClickHouse/blob/master/docs/mkdocs.yml

Reply:
Related to #1467

Reply:
Clickhouse seems to have written some python scripts to deal with various problems, instead of using configuration files.

In the new version of the website, we also have ways to reduce the content of the configuration file.